storm-client/src/jvm/org/apache/storm/pacemaker/codec/ThriftNettyClientCodec.java

```diff
@@ -72,7 +72,7 @@ protected void initChannel(Channel ch) throws Exception {
                 throw new RuntimeException(e);
             }
         } else {
-            client.channelReady(ch);
+            // no work for AuthMethod.NONE
```

Comment:
Consider removing the else block entirely

Reply:
My personal preference is to leave it so we know AuthMethod.NONE was not skipped by accident.

Reply:
Ok, makes sense.